packages/ember-htmlbars/tests/helpers/component_test.js

```diff
@@ -1,398 +0,0 @@
-import Controller from 'ember-runtime/controllers/controller';
-import { set } from 'ember-metal/property_set';
-import { get } from 'ember-metal/property_get';
-import run from 'ember-metal/run_loop';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-import ComponentLookup from 'ember-views/component_lookup';
-import EmberView from 'ember-views/views/view';
-import Component from 'ember-views/components/component';
-import compile from 'ember-template-compiler/system/compile';
-import computed from 'ember-metal/computed';
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, owner;
-
-QUnit.module('ember-htmlbars: {{#component}} helper', {
-  setup() {
-    owner = buildOwner();
-
-    owner.registerOptionsForType('template', { instantiate: false });
-    owner.register('component-lookup:main', ComponentLookup);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-  }
-});
-
-QUnit.test('component helper with bound properties are updating correctly in init of component', function() {
-  owner.register('component:foo-bar', Component.extend({
-    init: function() {
-      this._super(...arguments);
-
-      equal(get(this, 'location'), 'Caracas', 'location is bound on init');
-    }
-  }));
-  owner.register('component:baz-qux', Component.extend({
-    init: function() {
-      this._super(...arguments);
-
-      equal(get(this, 'location'), 'Loisaida', 'location is bound on init');
-    }
-  }));
-  owner.register('template:components/foo-bar', compile('yippie! {{location}} {{yield}}'));
-  owner.register('template:components/baz-qux', compile('yummy {{location}} {{yield}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    dynamicComponent: computed('location', function() {
-      var location = get(this, 'location');
-
-      if (location === 'Caracas') {
-        return 'foo-bar';
-      } else {
-        return 'baz-qux';
-      }
-    }),
-    location: 'Caracas',
-    template: compile('{{#component view.dynamicComponent location=view.location}}arepas!{{/component}}')
-  }).create();
-
-  runAppend(view);
-  equal(view.$().text(), 'yippie! Caracas arepas!', 'component was looked up and rendered');
-
-  run(function() {
-    set(view, 'location', 'Loisaida');
-  });
-  equal(view.$().text(), 'yummy Loisaida arepas!', 'component was updated and re-rendered');
-
-  run(function() {
-    set(view, 'location', 'Caracas');
-  });
-  equal(view.$().text(), 'yippie! Caracas arepas!', 'component was updated up and rendered');
-});
-
-QUnit.test('component helper with unquoted string is bound', function() {
```

Comment:
covered by the existing, slightly modified `@test it can render a basic component with a dynamic component name argument`